Frameworks/GLKit/GLKEffect.mm

```diff
@@ -522,9 +522,9 @@ - (BOOL)prepareShaders {
 
         NSTraceVerbose(TAG, @"For shader named: %@", self.shaderName);
         NSTraceVerbose(TAG, @"---[ VERTEX SHADER ]------------------------------------------------------------");
-        NSTraceVerbose(TAG, p.vertexShader);
```

Comment:
@bviglietta Agreed! The format annotations were totally missing/disabled, though, so.. it never knew :grin: